.travis.yml

```diff
@@ -1,32 +1,22 @@
-sudo: required
-
-dist: trusty
-
 language: python
+dist: trusty
+sudo: false
 
-python:
-  - 2.7
-  - 3.5
+matrix:
+  include:
+  - python: 2.7
+    env: TOX_ENV=py27
+  - python: 3.5
+    env: TOX_ENV=py35
+  - python: 2.7
+    env: TOX_ENV=flake8
 
 before_install:
-  - sudo apt-get install -y python-setuptools
   - npm install -g typescript
   - tsc --module amd --noImplicitAny --outdir datalab/notebook/static datalab/notebook/static/*.ts
-  - pip install -U pip
-  - pip install -U setuptools
-  - if [[ $TRAVIS_PYTHON_VERSION == 2* ]]; then pip install -U google-cloud-dataflow==0.5.5; fi
-  - pip install .
-  - pip install solutionbox/structured_data/
-  - pip install solutionbox/image_classification/  
-  - pip install -U tensorflow==1.0.0
-  - pip install -U protobuf==3.1.0  #v3.2.0 has the "python: double free or corruption" bug.
-  - pip install flake8
-
-before_script:
-  - flake8 .
-
+  - pip install --upgrade pip setuptools tox coveralls
```

Comment:
why do we have two sets of dependencies 
1) install things in this list
2) install things in tox.ini

Can we have just one list?